docs/configuration.md

```diff
@@ -211,13 +213,38 @@ creating trades.
 Once you will be happy with your bot performance, you can switch it to
 production mode.
 
+### Dynamic Pairlists
+
+Dynamic pairlists allow configuration of the pair-selection. 
+Basically, the bot runs against all pairs (with that stake) on the exchange, and a number of assets (`number_assets`) is selected based on different criteria.
+
+By *default*, a Static Pairlist is used (configured as `"pair_whitelist"` under the `"exchange"` section of this configuration).
+
+#### Available Pairlist methods
+
+* `"StaticPairList"`
+  * uses configuration from `exchange.pair_whitelist` and `exchange.pair_blacklist`
+* `"VolumePairList"`
+  * Formerly available as `--dynamic-whitelist [<number_assets>]`
+  * Selects `number_assets` top pairs based on `sort_key`, which can be one of `askVolume`, `bidVolume` and `quoteVolume`, defaults to `quoteVolume`.
+
```

Comment:
yes ... that's the main reason. the other 2 options would have been there before, but kindof hidden ... 
in theory, it could also default to nothing and fail if it's not explicitly there - but i don't really wanna change the default behaviour without particular reason other than because i think another sorting-strategy works better - which may or may not be correct depending on the strategy.